third_party/move/evm/move-to-yul/tests/Vectors.exp

```diff
@@ -642,19 +642,19 @@ object "test_A2_Vectors_test_borrow_mut" {
                     $t11 := 90
                     // $t12 := borrow_field<Vectors::S>.x($t10)
                     $t12 := $t10
-                    // write_ref($t12, $t11)
+                    // write_ref($t11, $t12)
                     $StoreU128($t12, $t11)
                     // $t13 := false
                     $t13 := false
                     // $t14 := borrow_field<Vectors::S>.y($t10)
                     $t14 := $IndexPtr($t10, 24)
-                    // write_ref($t14, $t13)
+                    // write_ref($t13, $t14)
                     $StoreU8($t14, $t13)
                     // $t15 := 1028
                     $t15 := 1028
                     // $t16 := borrow_field<Vectors::S>.z($t10)
                     $t16 := $IndexPtr($t10, 16)
-                    // write_ref($t16, $t15)
+                    // write_ref($t15, $t16)
```

Comment:
Wouldn't need to look through all those diffs without this change. What is the reason to do this in this PR? Was it technically needed?

Reply:
Yes, without this, we would need to handle `WriteRef` instructions specially (due to reversed order) in a few places (else we would reorder instructions in the incorrect order, causing more flushes).